docs/api/session.md

```diff
@@ -46,7 +46,7 @@ The `session` module has the following properties:
 
 ### `session.defaultSession`
 
-Returns the default session object of the app.
+A `Session` object, the default session object of the app.
```

Comment:
This property is read-only right? If so, should document it as:

Returns the `Session` object for the default session of the app.